src/components/Links/EtherScanLink.spec.js

```diff
@@ -0,0 +1,53 @@
+import { render } from '@testing-library/react'
+
+jest.mock('@apollo/client', () => ({
+  __esModule: true,
+  ...jest.requireActual('@apollo/client'),
+  useQuery: jest.fn()
+}))
+import { useQuery } from '@apollo/client'
+
+import EtherScanLink from './EtherScanLink'
+import { StaticRouter } from 'react-router-dom'
+
+describe('EtherScanLink', () => {
```

Comment:
Liking these component tests. Maybe write in the readme best practices for testing components so we can keep them at a similar standard, especially for newcomers